IncludeToolbox/Commands/IncludeWhatYouUse.cs

```diff
@@ -240,7 +240,7 @@ private string RunIncludeWhatYouUse(string fullFileName, EnvDTE.Project project,
                 process.StartInfo.Arguments += " -DM_X64 -DM_AMD64 ";// TODO!!!
 
                 // Clang options
-                process.StartInfo.Arguments += "-w -x c++ -std=c++14 -fcxx-exceptions -fexceptions -fms-compatibility -fms-extensions -fmsc-version=1900 -Wno-invalid-token-paste "; // todo fmsc-version!
+                process.StartInfo.Arguments += "-w -x c++ -std=c++14 -fcxx-exceptions -fexceptions -fms-compatibility -fms-extensions -fmsc-version=1900 -ferror-limit=0 -Wno-invalid-token-paste "; // todo fmsc-version!
```

Comment:
Add the option.

It might be nice to allow these to be specified in the properties (while defaulting to these). That might serve to "fix" the TODO here.